docs/DeveloperGuide.md

```diff
@@ -1543,7 +1544,7 @@ testers are expected to do more *exploratory* testing.
       persons have the assignment you wish to give using the `show` command.
 
    2. Test case: `giveall m/CS2100 d/Assignment 3 by/ 11/11/2021` <br>
-      Expected: All persons in the module "CS2100" have "Assignment 1" with due date "10/10/2021" added into their assignment list.
+      Expected: All persons in the module "CS2100" have "Assignment 1" with due date "11/10/2021" added into their assignment list.
```

Comment:
I think you missed out assignment 1 should be 3

Reply:
Good catch!